spec/rails_helper.rb

```diff
@@ -69,4 +69,19 @@
   config.include Devise::TestHelpers, type: :controller
   config.include SpecHelpers
   config.include Delorean
+
+  if ENV['RSPEC_TASK'] != 'spec:nofeatures'
```

Comment:
Sure we can do that, what do you think about moving the require of `capybara_helper` into `rails_helper`? The `require capybara_helper` gives the impression those configurations are only applied to the feature tests, when they are really affect all loaded tests.